src/snapred/backend/dao/GroupPeakList.py

```diff
@@ -0,0 +1,13 @@
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from snapred.backend.dao.DetectorPeak import DetectorPeak
+
+
+class GroupPeakList(BaseModel):
+    """Class to hold diffraction peak positions of one group"""
+
+    peaks: List[DetectorPeak]
+    groupID: int
+    spectrumNumber: Optional[int]
```

Comment:
After grouping, there is just one spectrum per group, correct? Then why do we need both `groupID` and `spectrumNumber`? Perhaps a comment would help.

Reply:
My understanding is that `spectrumNumber` and `groupID` are not guaranteed to be related to one another.  But actually, I don't think we even need `spectrumNumber`, since the algorithms never call for it